x.py

```diff
@@ -267,9 +267,14 @@ def download_package(bindir: str) -> None:
     env = os.environ.copy()
     env.update({
         "CGO_CFLAGS": f"-I{rocksdb}",
-        "CGO_LDFLAGS": f"-L{rocksdb_lib} -L{zlib} -L{z4lib} -L{snappy_lib}",
+        "CGO_LDFLAGS": f"-L{rocksdb_lib} -L{zlib} -L{z4lib} -L{snappy_lib} -L/lib64 -L/usr/lib64 -lstdc++",
+        "LD_LIBRARY_PATH": f"/lib64:/usr/lib64:{env.get('LD_LIBRARY_PATH', '')}"
```

Comment:
Could you explain this change? Otherwise we need to revert it.

Reply:
Hello @PragmaTwice please ignore the changes in x.py until I'm done fixing the ci that's failing on my fork.